cpp/ps0002/ps0002_sol.cpp

```diff
@@ -4,15 +4,17 @@ using namespace std;
 
 int main()
 {
-   int residents = 23 ;
-   int alternative = 9 ;
-   short members = 6 ;
-   double price = 5.5 ;
-   char ahmed = 'a' ;
-   long births = 1000000000  ;
-   double division = 3.5 ;
-   long bill = 1234560 ;
-   long age = 12345664 ;
+   unsigned short residents = 23; // 0.5/0.75
+   unsigned short alternative = 9; // 0.5/0.75
+   unsigned short members = 6; // 0.5/0.75
+   float price = 5.5; // 0.5/0.75
+   char first_letter = 'a'; // 0.5/0.75
+   unsigned int births = 1000000000; // 0.75/0.75
+   long double division = 3.1428571428665466; // 0.5/0.75
+   float bill = 1234560; // 0.5/0.75
+   unsigned int age = 12345664; // 0.5/0.75
+   // total 4.75/6.75
 
 return 0;
 }
+// good work 👍 7/10

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code